Creating the color swap file, I hope it works

teylorchapman opened the pull request:
Reused the sprite swap code, but changed everything from sprite to color, I hope it works

teylorchapman committed:
Creating the color swap file, I hope it works 

Reused the sprite swap code, but changed everything from sprite to color, I hope it works

Oggal commented on Assets/Scripts/ColorSwap.cs:
```diff
@@ -0,0 +1,19 @@
+public class ColorSwap : MonoBehaviour
+{
+
+    [SerializeField] ColorRenderer target;
```
This should still be a spritRenderer, and can we make it an array of them?

Oggal commented on Assets/Scripts/ColorSwap.cs:
```diff
@@ -0,0 +1,19 @@
+
+    public void swapColor(int i)
+    {
+        if (!target)
+            return;
+        target.color = colors[i];
```
Change this into 

foreach( SpriteRenderer t in target)
{ t.color = colors[i]; }

That'll loop through each renderer added to the array of them. The medium thunder clouds use 2 Sprite renders, so this will make setting them up easier

Oggal reviewed (changes requested):
If the comments aren't clear, let me know. This is wonderful and almost perfect!

teylorchapman committed:
Update ColorSwap.cs

Updated the SpriteRenderer again, made it an array, and update the script to swap through the colora

Oggal commented on Assets/Scripts/ColorSwap.cs:
```diff
@@ -0,0 +1,18 @@
+    public void swapColor(int i)
+    {
+        foreach( SpriteRenderer t in target)
+        t.color = colors[i];
```
Should indent this one more,
Looks great though!!

Oggal commented on Assets/Scripts/ColorSwap.cs:
```diff
@@ -0,0 +1,18 @@
+
+    [SerializeField] Color[] colors;
+    // Start is called before the first frame update
+
```
Remove blank line

Oggal reviewed (approved):
Only concerns I have are style corrections!
Looks great!

teylorchapman committed:
ColorSwap with style changes :)
